packages/components/src/components/overlay/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FC, MouseEvent } from 'react';
+import {
+  Drawer as ChakraDrawer,
+  DrawerOverlay as ChakraDrawerOverlay,
+  DrawerContent as ChakraDrawerContent,
+  DrawerFooter as ChakraDrawerFooter,
+  DrawerBody as ChakraDrawerBody,
+  DrawerHeader as ChakraDrawerHeader,
+  DrawerProps,
+  ModalOverlayProps,
+  DrawerContentProps,
+  ModalFooterProps,
+  ModalBodyProps,
+  ModalHeaderProps,
+  forwardRef,
+  useModalContext,
+} from '@chakra-ui/react';
+import { DrawerStaticMembers } from './types';
+import { CloseButton, CloseButtonProps } from '@components';
+import { callAllHandlers } from '@chakra-ui/utils';
+
+
+export const Drawer: FC<DrawerProps> & DrawerStaticMembers = ({ children, ...rest }) => (
+  <ChakraDrawer {...rest}>
+    {children}
+  </ChakraDrawer>
+);
+
+const DrawerOverlay: FC<ModalOverlayProps> = (props) => <ChakraDrawerOverlay {...props} />;
+
+const DrawerContent: FC<DrawerContentProps> = ({ children, ...rest }) => (
```

Comment:
Could we simplify each static member that doesn't have any overrides to:

`Drawer.Content = ChakraDrawerContent`
...

Reply:
Done 👍🏼